src/ol/source/vector.js

```diff
@@ -610,11 +610,12 @@ ol.source.Vector.prototype.getClosestFeatureToCoordinate = function(coordinate,
  *
  * This method is not available when the source is configured with
  * `useSpatialIndex` set to `false`.
+ * @param {ol.Extent=} opt_extent Destination extent.
```

Comment:
Hm. Do we expose reusable arrays elsewhere in the API? As a user, I'd probably never pass that `opt_extent`. But maybe that's just me.

If we decide to expose that to the API, the option should at least be better documented, so users know what "Destination extent" means exactly.